velox/connectors/utils/StringFormatter.h

```diff
@@ -147,7 +161,8 @@ struct DefaultFormatter : public StringFormatter {
     } else if constexpr (std::is_same_v<T, StringView>) {
       ss << t.str();
     } else if constexpr (std::is_same_v<T, Timestamp>) {
-      ss << t.toString();
+      TimestampToStringOptions options{.precision=TimestampPrecision::kMilliseconds};
```

Comment:
Is this parameter in flink configurable? And does toString respect to timezone in flink?

Reply:
It's not configurable in flink, but to specify it by `DataType` when define the field, such as `a timestamp(3)`.  Flink's `toString` respect to timezone but here it is not, I'll try to implement it.